src/css/embed.scss

```diff
@@ -252,31 +254,127 @@ $brexit-control-button-size: 28px;
 
 
 /****** BEGIN Slider ******/
+.brexit--slider {
+    background-color: transparent;
+    padding: 0;
+
+    .brexit__nav-control--prev {
+        left: 0;
+    }
+    .brexit__nav-control--next {
+        right: 0;
+    }
+}
 .brexit__slider-wrapper {
     overflow: hidden;
 }
 .brexit__slider {
-    @include clearfix;
-    position: relative;
-    left: -100%;
-    // currently hard-coded to 3 sliders
-    // TODO: variable number of sliders
+    display: flex;
+    flex-direction: row;
     width: 300%;
+
+    transform: translateX(-33.333%);
+    transition-property: transform;
+    transition-duration: 0.3s;
 }
 .brexit__slider--left {
-    left: 0;
+    transform: translateX(0);
 }
 .brexit__slider--right {
-    left: -200%;
+    transform: translateX(-66.666%);
 }
 .brexit__slider__item {
-    float: left;
-    // currently hard-coded to 3 sliders
-    // TODO: variable number of sliders
-    width: 33.3%;
+    flex-grow: 1;
+    flex-basis: 0;
+    margin: 0 $brexit-control-button-size*0.5;
+    padding: 6px 21px;
+    background-color: colour(news-main-1);
+    width: 1px;
+}
+/****** END Slider ******/
+
+/****** BEGIN Two-sided ******/
+
+.brexit__two-sided {
+    display: flex;
+    flex-direction: column;
+}
+.brexit__two-sided__row {
+    flex-grow: 1;
+    flex-shrink: 1;
+    flex-basis: auto;
+}
+.brexit__two-sided__question {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    .brexit__header {
+        padding-bottom: 0;
+    }
+}
+.brexit__two-sided__top, .brexit__two-sided__bottom {
+    display: flex;
```

Comment:
also for a separate PR, but let's look at refactoring to pull out some of these common flex-box setups into mixins